rts/lra/client/src/main/java/io/narayana/lra/client/NarayanaLRAClient.java

```diff
@@ -324,12 +326,13 @@ public URI startLRA(URI parentLRA, String clientID, Long timeout, ChronoUnit uni
             // validate the HTTP status code says an LRA resource was created
             if (isUnexpectedResponseStatus(response, Response.Status.CREATED)) {
                 String responseEntity = response.hasEntity() ? response.readEntity(String.class) : "";
-
+                String logMsg = LRALogger.i18nLogger.error_lraCreationUnexpectedStatus(response.getStatus(), responseEntity);
+                String errorMsg = String.format("%s (%s)", logMsg);
```

Comment:
Insufficient parameters for format string. This mistake is also done elsewhere so I will not repeat this review comment but it needs fixing in all places.